qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1520A.py

```diff
@@ -83,3 +110,38 @@ def _get_capacitance(self) -> Tuple[float, float]:
             raise ValueError("Result format not supported.")
 
         return float(parsed[0]["value"]), float(parsed[1]["value"])
+
+    def _set_phase_compensation_mode(self, mode: constants.ADJ.Mode) -> None:
+        msg = MessageBuilder().adj(chnum=self.channels[0], mode=mode)
+        self.write(msg.message)
+
+    def _query_phase_compensation(self):
+        with self.root_instrument.timeout.set_to(self.time_out_phase_compensation):
+            msg = MessageBuilder().adj_query(chnum=self.channels[0],mode=constants.ADJQuery.Mode.MEASURE)
+            response = self.ask(msg.message)
+        return constants.ADJQuery.Response(int(response))
+
+    def _clear_freq_list(self,mode: constants.CLCORR.Mode):
+        msg = MessageBuilder().clcorr(chnum=self.channels[0], mode=mode)
+        self.write(msg.message)
+
+    def _set_frequency_for_correction_data_measurement(self, freq: int):
+        msg = MessageBuilder().corrl(chnum=self.channels[0], freq=freq)
+        self.write(msg.message)
+
+    def _get_frequency_for_correction_data_measurement(self):
+        msg = MessageBuilder().corrl_query(chnum=self.channels[0])
+        response = self.ask(msg.message)
+        return response
+
+    def _perform_correction_data_measurement(self):
+        msg = MessageBuilder().corr_query(chnum=self.channels[0], corr=constants.CalibrationType.OPEN)
+        response = self.ask(msg.message)
+        return response
+
+    def abort(self):
```

Comment:
is this a general-purpose command? if so, i believe that it deserves its own little section in the example notebook. importantly, users should realize that this command can't be used to abort a long `.ask` (so i actually wonder, when can it be used?)

Reply:
You are right. I also think this command is more or less redundant. I cannot think of a use case where it will be useful mainly because the VISA will be occupied and will not be able to send abort command. However since the driver supports it and the user requested it,  I have implemented the command. I have added a docstring describing the functionality of the command.